pom.xml

```diff
@@ -535,6 +538,24 @@
             </dependency>
             <!--log4j end-->
 
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.aliyun.oss</groupId>
+                <artifactId>aliyun-sdk-oss</artifactId>
+                <version>${aliyun.oss.version}</version>
+            </dependency>
```

Comment:
The root pom declare global general dependencies, such as oss, s3, etc., which are the dependencies that only need to be used by streampark-storeage module. so, it is recommended to define it in streampark-storeage module

Reply:
addressed.

Reply:
I see you just move properties to `streampark-storage` module.

As I understand, @wolfboys  means move these dependencies to `streampark-storage` module, not just move properties. Root pom shouldn't add these dependencies, they should be added in `streampark-storage` module.

Reply:
> I see you just move properties to `streampark-storage` module.
> 
> As I understand, @wolfboys means move these dependencies to `streampark-storage` module, not just move properties. Root pom shouldn't add these dependencies, they should be added in `streampark-storage` module.

yes